2-collectors/scala-stream-collector/core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/model.scala

```diff
@@ -130,6 +136,7 @@ package model {
   final case class CollectorConfig(
     interface: String,
     port: Int,
+    ssl: Boolean,
```

Comment:
Would it be possible to make this parameter optional in the configuration file and to give it a default value "false" if absent? It would make TechOps' work easier for R117, in that it wouldn't require to update the configuration files when people don't want to activate SSL.